### PR TITLE
Navbar & Sidebar Components

### PR DESCRIPTION
**Pull request by revett**
### Problem

The navbar and sidebar on BR pages are currently in the ERB templates.

### Solution

Move to within React components, for the Insights page.

### Notes

- https://trello.com/c/MyErzcJl/248-br-port-insights-ui-to-react-components

### CODE REVIEW

File: source/jsx/brands/navigation/navigation.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```
Comment (benjdlambert) [inline source/jsx/brands/navigation/navigation.jsx]: Filename. Won't compile properly under `linux`

Comment (revett) [inline source/jsx/brands/navigation/navigation.jsx]: What do you mean?

**Comment by benjdlambert** [comment]
The filenames are case insensitive on Mac, but when you've pushed to github you can see they are lowercase.

It might be a pain in the arse to get them into Git with the fixed filename, but it will need doing before merge..

:cake: after fixing the filenames

**Comment by benjdlambert** [comment]
Filename is `navigation/navigation.jsx`

But you require `navigation/Navigation.jsx`

**Comment by revett** [comment]
Closing due to fuck up w/ filenames 

cc/ @benjdlambert

**Pull request closed without merging**